### PR TITLE
Add Deprecated and Beta folders

### PR DESCRIPTION
**Pull request by grellyd**
## Summary

This MR adds the `/deprecated` and `/beta` folders for the scim-examples repo, and adds the deprecated terraform deployment to it.

## How to Test

Does the documentation make sense? Do the documentation links work? Are there spelling or grammatical errors?

### CODE REVIEW

File: deprecated/aws-terraform/README.md [hunk 1]
```diff
@@ -2,13 +2,15 @@
 
 This document describes deploying the 1Password SCIM bridge to your Amazon Web Service cloud services using Terraform.
 
+**NOTE** This deployment has been deprecated in favour of the [aws-ecsfargate-terraform](../aws-ecsfargate-terraform) deployment.
```
Comment (grellyd) [inline deprecated/aws-terraform/README.md:5]: Ah looks like this link isn't quite right. 

- [ ] Fix link destination

File: deprecated/aws-terraform/README.md [hunk 2]
```diff
@@ -2,13 +2,15 @@
...
+
 ## Preparing
 
 This is only an example of how you could deploy the 1Password SCIM bridge to your existing AWS infrastructure. As an advanced form of deployment, this AWS/Terraform example can be integrated into your existing infrastructure. Feel free to modify the deployment to suit your specific needs.
 
 If you find that you are not using a majority of the services listed in this documentation within your AWS account - for instance, you have an external DNS provider, or you are providing your own certificates through a third-party service - consider deploying through [Kubernetes](https://github.com/1Password/scim-examples/tree/master/kubernetes/README.md) using AWS Elastic Kubernetes Service (EKS). The 1Password SCIM bridge is not a resource-intensive service, and a full AWS deployment may include more infrastructure than you require, unless it fits neatly into your existing setup.
 
-To continue, please ensure you've read through [PREPARATION.md](/PREPARATION.md) before beginning deployment.
+To continue, please ensure you've read through [PREPARATION.md](../PREPARATION.md) before beginning deployment.
```
Comment (grellyd) [inline deprecated/aws-terraform/README.md:13]: Same here.

- [ ] Fix link destination

File: deprecated/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Deprecated Deployments
+
+This folder contains deprecated versions of 1Password SCIM bridge deployments. 
+
+At the time of deprecation, these deployments are still fully functional, but may no longer be updated and will eventually be removed.
+
+## Deployments
+ - aws-terraform (2020-12-21)
```
Comment (laughingman-hass) [inline deprecated/README.md:8]: Can we turn this one to a link as well please

**Review by laughingman-hass — DISMISSED** [review]
small nitpick, but otherwise looks good with the corrections of the links you've already highlighted

File: deprecated/README.md (hunk 3 above)
Comment (grellyd) [reply deprecated/README.md:8]:
```suggestion
 - [aws-terraform](./aws-terraform) (2020-12-21)
```

File: deprecated/aws-terraform/README.md (hunk 1 above)
Comment (grellyd) [reply deprecated/aws-terraform/README.md:5]:
```suggestion
**NOTE** This deployment has been deprecated in favour of the [aws-ecsfargate-terraform](../../aws-ecsfargate-terraform) deployment.
```

File: deprecated/aws-terraform/README.md (hunk 2 above)
Comment (grellyd) [reply deprecated/aws-terraform/README.md:13]:
```suggestion
To continue, please ensure you've read through [PREPARATION.md](../../PREPARATION.md) before beginning deployment.
```

**Review by laughingman-hass — APPROVED** [review]

**Pull request merged**